shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -316,16 +316,15 @@ activeSlotLog f = (fromIntegral $ unActiveSlotLog f) / fpPrecision
 data Globals = Globals
   { epochInfo :: !(EpochInfo Identity)
   , slotsPerKESPeriod :: !Word64
+    --
+    --   The nonce leakage parameter τ from the paper
```

Comment:
Is this tau? I thought it was tau * k?

Reply:
I _think_ it is just tau. There are two tau's in the paper, this one is _not_ the "speed coefficient", but the number of slots prior to the end of the epoch that are leaked to the adversary. so the one in the scripty-F^{tau, r}. but maybe I am wrong!

Reply:
Aesthetics: If you remove the first empty comment line and add a pipe | before "The" it would be clearer which field this is documenting (and `haddock` will notice it).

I think Jared is right. Separately, ChainGrowth(τ,s) is roughly "at least `τ * s` blocks on a chain in every `s` slot window"; so we choose the params such that `k > τ * s`. But the Praos paper at least does use τ for two different things.

Reply:
Hmm. This is not obvious.

I think `stabilityWindow` is actually `s` (doubled, for HardFork-ish stable stability) from our chosen ChainGrowth(τ,s) instance, after setting the speed coefficient `τ = k/s`.

On the other hand, it's `randomnessStabilisationWindow` that corresponds to the nonce leakage parameter.

I'd guess they're values are probably related, even though I don't recall the paper discussing that.

Summary: I don't think there's a simple phrase to characterize `stabilityWindow`. Maybe "the window size in which our chosen Chain Growth property guarantees at least `k` blocks".

Reply:
Thank you for noticing the formatting! Would you suggest a better way to connect `stabilityWindow` to the paper, or do you think it is clear enough?

Reply:
there was a delay on me seeing your second comment here :) thank you, I'll reword accordingly.